.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,25 @@
+name: Build wheels
+
+on:
+  push:
+    branches: [master]
+    tags: ['*']
+    paths-ignore:
+    - 'docs/**'
+    - '**/*.rst'
+    - '**/*.md'
+    - .gitignore
+    - .travis.yml
+    - CREDITS
+    - LICENSE
+
+jobs:
+  build-wheels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - name: Trigger script
+        run: ./misc/trigger_wheel_build.sh
```

Comment:
Does this work?
If not, do we know who has perms to make WHEEL_PUSH_TOKEN? https://github.com/python/mypy/blob/7b9aa4cfe2cb57fa3f90a917dcd02620986bee79/misc/trigger_wheel_build.sh#L6

Reply:
Can't imagine that it would work. I hopefully have the creds somewhere, let me look

Reply:
This might be helpful? https://docs.github.com/en/actions/reference/encrypted-secrets

Reply:
Alright, I have added a token with the secret name `WHEELS_PUSH_TOKEN`